docs/developers/plugins.md

```diff
@@ -2,16 +2,20 @@
 
 Plugins are the most efficient way to customize or change the default behavior of a chart. They have been introduced at [version 2.1.0](https://github.com/chartjs/Chart.js/releases/tag/2.1.0) (global plugins only) and extended at [version 2.5.0](https://github.com/chartjs/Chart.js/releases/tag/2.5.0) (per chart plugins and options).
 
-## Popular Plugins
+## Popular Chart Type Plugins
```

Comment:
Since charts are not proper plugins, we could move this new section in the [charts.md](/chartjs/Chart.js/blob/master/docs/developers/charts.md) file before the [Dataset Controller Interface](/chartjs/Chart.js/blob/master/docs/developers/charts.md#dataset-controller-interface) section?